lib/index.js

```diff
@@ -0,0 +1,131 @@
+"use strict";
```

Comment:
`lib/` shouldn't be committed to github. Can you add a line in .gitignore?

Reply:
I was using the fork directly in some  project.. but i will .gitignore it